### PR TITLE
Update sample code to v3

### PR DESCRIPTION
**Pull request by antspriggs**

### CODE REVIEW

**Comment by tonycchung** [comment]
i think how the response is parsed needs to be updated as well

**Comment by tonycchung** [comment]
add a CHANGELOG.md and should be good

**Review by tonycchung — APPROVED** [review]

File: templates/index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 
 <body class="main">
-  <a href="https://idp-staging.idmeinc.net/oauth/authorize?client_id=aeab0d4282e2127f900075d9673aeca5&redirect_uri=http://localhost:5000/callback&response_type=code&scope=military" class="idme-btn">
+  <a href="https://api.id.me/oauth/authorize?client_id=aeab0d4282e2127f900075d9673aeca5&redirect_uri=http://localhost:5000/callback&response_type=code&scope=military" class="idme-btn">
```
Comment (zachallett) [inline templates/index.html]: which client credentials are those?

**Review by zachallett — CHANGES_REQUESTED** [review]

File: templates/index.html (hunk 1 above)
Comment (antspriggs) [reply templates/index.html]: Not sure. Is there a way I can search that in admin?

Comment (zachallett) [reply templates/index.html]: In that case, we should remove it with instructions on how to add their own client credentials

**Review by zachallett — APPROVED** [review]

**Pull request merged**